libflux/src/libflux/lib.rs

```diff
@@ -1,4 +1,8 @@
 #![cfg_attr(feature = "strict", deny(warnings))]
+// XXX: phummer (2 Dec 2019) - These lints should be removed one at a time until
+// the line is entirely removed. If lint still must be ignored, it should be
+// at a more specific module or file level.
+#![allow(clippy::useless_format, clippy::cmp_owned, clippy::while_let_loop, clippy::single_char_pattern, clippy::chars_last_cmp, clippy::chars_next_cmp, clippy::unnecessary_operation, clippy::new_without_default, clippy::wrong_self_convention, clippy::try_err, clippy::ptr_offset_with_cast, clippy::clone_on_copy, clippy::useless_let_if_seq, clippy::implicit_hasher, clippy::ptr_arg, clippy::large_enum_variant, clippy::single_match, clippy::unnecessary_fold, clippy::not_unsafe_ptr_arg_deref, clippy::len_zero, clippy::or_fun_call, clippy::needless_return, clippy::collapsible_if, clippy::module_inception, clippy::many_single_char_names, clippy::redundant_field_names, clippy::unknown_clippy_lints)]
```

Comment:
Can this line be broken up into multiple lines?

Reply:
I recall some pickiness from the compiler about multiple lines, but a small test seems to indicate it can. Change inbound.